src/wexond/renderer/app/components/Toolbar/index.tsx

```diff
@@ -53,46 +52,57 @@ export const Toolbar = observer(() => {
   }
 
   const onKeyboardPress = () => {
-    barcoKeyBoardController.toggle()
+    ipcRenderer.send(VirtualKeyboardEvent.TOGGLE)
+  }
+
+  const buttons: JSX.Element = (<Buttons>
+    <BrowserActions />
+    {store.updateInfo.available && (
+      <ToolbarButton icon={icons.download} onClick={onUpdateClick} />
+    )}
+    {store.extensions.browserActions.length > 0 && <Separator />}
+    {!isWindow && (
+      <BrowserAction
+        size={18}
+        style={{ marginLeft: 0 }}
+        opacity={0.54}
+        data={{
+          badgeBackgroundColor: 'gray',
+          badgeText: blockedAds > 0 ? blockedAds.toString() : '',
+          icon: icons.shield,
+          badgeTextColor: 'white',
+        }}
+      />
+    )}
+    <ToolbarButton
+      disabled={false}
+      size={TOOLBAR_ICON_HEIGHT}
+      icon={icons.home}
+      onClick={onHomePress}
+    />
+    <ToolbarButton
+      disabled={false}
+      size={TOOLBAR_ICON_HEIGHT}
+      icon={icons.keyboard}
+      onClick={onKeyboardPress}
+    />
+  </Buttons>)
+
+  if (data.disableTabs) {
+    return (
+        <StyledToolbar isHTMLFullscreen={store.isHTMLFullscreen}>
+          <NavigationButtons />
+          {buttons}
+        </StyledToolbar>
+    )
   }
 
   return (
       <StyledToolbar isHTMLFullscreen={store.isHTMLFullscreen}>
         <NavigationButtons />
         <Tabbar />
         <Find />
```

Comment:
The only difference between the two renders are `<Tabbar />` and `<Find />`
It would have been much easier and less awkward to condtionnaly render them instead of having two returns and all the buttons stored in a variable

```
{!data.disableTabs && (<>
    <Tabbar />
    <Find />
</>)}
```